provider/cmd/tests/__main__.py

```diff
@@ -0,0 +1,9 @@
+"""An AWS Python Pulumi program"""
+
+# Append provider root to path.
+import sys
+
+sys.path.append("../pulumi-resource-nuage")
```

Comment:
You mentioned that this could be potentially fixed by using Poetry to install the package (using a path dependency)